MovieQueueManagerGUI/MovieQueueManagerGUI.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
So here you update from 4 to 12, but what exactly is it that you're updating?  What's a schema? When I go to the URL, it just says that the resource has been moved.